docs/data-cookbooks/throttling-queued-executions.md

```diff
@@ -83,12 +80,9 @@ Follow the instructions to [re-deploy your Cumulus application](./../deployment/
 
 ```json
 {
-  "meta": {
-    "queues": {
-      "backgroundJobQueue": "<backgroundJobQueue_SQS_URL>"
-    },
+  "cumulus_meta": {
     "queueExecutionLimits": {
-      "backgroundJobQueue": 5
+      "<backgroundJobQueue_SQS_URL>": 5
```

Comment:
I don't see your comment about adding things to `cumulus_meta` during workflows - but to be clear `cumulus_meta` would not be changed during workflows. This snippet is just showing how the workflow message template is changed **at deploy time** when you configure more queues with execution throttling. At runtime, `cumulus_meta` is only read from and not changed

Reply:
Right, I removed it because exactly what you said.